package.json

```diff
@@ -11,17 +11,16 @@
     "url": "https://github.com/feross/p2p-graph/issues"
   },
   "dependencies": {
-    "d3": "^3.5.6",
-    "debounce": "^1.0.0",
-    "debug": "^2.2.0",
-    "prettier-bytes": "^1.0.3"
+    "d3": "3.5.6",
+    "debounce": "1.0.0",
+    "debug": "2.2.0",
+    "prettier-bytes": "1.0.3"
```

Comment:
What's the reasoning behind this? Exact versions make it hard for `npm install` to dedupe dependencies. For example, if prettier-bytes 1.0.4 comes out, then the webtorrent.io will have to bundle both 1.0.3 and 1.0.4 because we depend on the latest version there.